files/es/learn/common_questions/what_are_browser_developer_tools/index.md

```diff
@@ -192,35 +192,35 @@ Esto te dará una ventana como la siguiente:
 
 Para ver qué sucede, intenta ingresar los siguientes fragmentos de código en la consola uno por uno (y luego presiona Intro):
 
-1. ```js
-    alert('hello!');
-    ```
-    
-2. ```js
-    document.querySelector('html').style.backgroundColor = 'purple';
-    ```
-
-3. ```js
-    const myWordmark = document.createElement('img');
-    myWordmark.setAttribute('src','https://blog.mozilla.org/press/wp-content/themes/OneMozilla/img/mozilla-wordmark.png');
-    document.querySelector('h1').appendChild(myWordmark);
-    ```
+```js
+alert('hello!');
+```
+
+```js
+document.querySelector('html').style.backgroundColor = 'purple';
+```
+
+```js
+const myWordmark = document.createElement('img');
+myWordmark.setAttribute('src','https://blog.mozilla.org/press/wp-content/themes/OneMozilla/img/mozilla-wordmark.png');
+document.querySelector('h1').appendChild(myWordmark);
+```
```

Comment:
The en-US versión is like this

```suggestion
1. ```js
    alert('hello!');
    ```
    
2. ```js
    document.querySelector('html').style.backgroundColor = 'purple';
    ```

3. ```js
    const myWordmark = document.createElement('img');
    myWordmark.setAttribute('src','https://blog.mozilla.org/press/wp-content/themes/OneMozilla/img/mozilla-wordmark.png');
    document.querySelector('h1').appendChild(myWordmark);
    ```
```
```

Reply:
Ah, I'll go fix that upstream too. I think it was a glitch in the HTML to Markdown conversion

Reply:
Submitted https://github.com/mdn/content/pull/20324, so we'll see if that lands.
Otherwise, you can add you suggestions and land this and it can be dealt with later